projects/packages/connection/src/class-manager.php

```diff
@@ -972,6 +972,20 @@ public function register( $api_endpoint = 'register' ) {
 
 		$this->get_tokens()->update_blog_token( (string) $registration_details->jetpack_secret );
 
+		if ( ! $registration_details->allow_inplace_authorization ) {
```

Comment:
I tend to have a slight preference for `empty()` since it will also handle the case of the property not being set.

Reply:
If `$registration_details->allow_inplace_authorization` is not set, this will generate a PHP Notice. Is there a reason that we don't use `$allow_inplace_authorization` here?